packages/TelegramClient-Core.package/TCCChat.class/instance/lastMessageId.st

```diff
@@ -0,0 +1,6 @@
+accessing
+lastMessageId
+
+	^ self messages
+		ifEmpty: [ self class defaultFromMessageId ]
```

Comment:
defaultFromMessageId is a bit of a strange name, I think it's fine but maybe something like defaultLastMessageId would fit better?